OrthoEvol/Orthologs/Blast/comparative_genetics.py

```diff
@@ -319,11 +355,11 @@ def get_orthologous_gene_sets(self, go_list=None):
                   correspond to the go_list index.
         """
 
-        if go_list is None:
+        if self.go_list is None:
```

Comment:
`go_list` is a parameter of this function and should be used that way.  I don't think this breaks any code, but it should be explicitely set either by the class initialization (BaseCG) or the method call (get_orth_g_s).

Reply:
An oversight here.